internal/main.go

```diff
@@ -0,0 +1,57 @@
+package main
```

Comment:
internals should never have a main package declaration, i'm bit confused about this package, internal should be private business logic that can be object to change and not importable by anyone outside of this project